### PR TITLE
Add story for TextAction

### PR DESCRIPTION
**Pull request by gustaveen**
## What 
Add stories for TextAction

https://user-images.githubusercontent.com/6661511/104480125-84821280-55c4-11eb-986c-aa7f7ce5f603.mov

### CODE REVIEW

File: src/Components/Actions/NumberAction/NumberAction.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    name: 'personalNumberApartment',
+  },
+  mask: 'Email',
+  minValue: '1',
+  placeholder: '2',
+  storeKey: 'householdSize',
+  tooltip: {
+    title:
+      'Exempelvis partner, barn eller roomies som ska täckas. De medförsäkrade måste bo tillsammans med dig för att täckas av din försäkring',
+    description: 'Antal försäkrade',
```
Comment (styrelius) [inline src/Components/Actions/NumberAction/NumberAction.stories.tsx:28]: I only get the `description` when I hover the question mark, not the `title` - isn't the `title` text the one we want?

image

Comment (styrelius) [reply src/Components/Actions/NumberAction/NumberAction.stories.tsx:28]: And the size of that image is insane, sorry 😳

File: src/Components/Actions/TextAction/TextAction.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+}
+
+export const PersonalNumber = Template.bind({})
+PersonalNumber.args = {
+  isTransitioning: false,
+  passageName: 'personalNumberApartment',
+  mask: 'PersonalNumber',
+  storeKey: 'personalNumber',
+  placeholder: 'ååmmdd-xxxx',
+}
```
Comment (styrelius) [inline src/Components/Actions/TextAction/TextAction.stories.tsx:34]: Shouldn't the `PersonalNumber` be a `NumberAction`?

**Review by styrelius — COMMENTED** [review]
Awesome! 🥳  Just two things 👇

File: src/Components/Actions/NumberAction/NumberAction.stories.tsx (hunk 1 above)
Comment (gustaveen) [reply src/Components/Actions/NumberAction/NumberAction.stories.tsx]: I copied the tooltip content from what we have in production today, but I accidentally switched places on title and description 😄 (if that caused the confusion) However the title is only used in mobile where you get a Modal 👍 


https://user-images.githubusercontent.com/6661511/104610184-1d2b9780-5684-11eb-929a-d023da715d75.mov

File: src/Components/Actions/TextAction/TextAction.stories.tsx (hunk 2 above)
Comment (gustaveen) [reply src/Components/Actions/TextAction/TextAction.stories.tsx:34]: We only use it for simple numbers like household size. For the same reason that we switched it in the ssn input in the checkout, you don't want to accidentally increase you personal number if pressing a number key for example :)

Comment (styrelius) [reply src/Components/Actions/TextAction/TextAction.stories.tsx:34]: AH, you are so right! 🧠

**Review by styrelius — APPROVED** [review]
NICE 😀

File: src/Components/Actions/NumberAction/NumberAction.stories.tsx (hunk 1 above)
Comment (styrelius) [reply src/Components/Actions/NumberAction/NumberAction.stories.tsx]: 👍

File: src/Components/Actions/TextAction/TextAction.stories.tsx (hunk 2 above)
Comment (gustaveen) [reply src/Components/Actions/TextAction/TextAction.stories.tsx:34]: I thought the exact same thing when making the stories so you're not alone 😄

**Pull request merged**